apps/docs/src/content/learn/advanced/webgpu.mdx

```diff
@@ -0,0 +1,192 @@
+---
+category: Advanced
+title: WebGPU and TSL
+order: 1
+---
+
+<Tip type="warning">
+  WebGPU support in Three.js is still in an early stage and is subject to change from Three.js
+  version to version.
+</Tip>
+<Tip type="warning">
+  We highly recommend targeting [version r171](https://github.com/mrdoob/three.js/releases/tag/r171)
+  onwards because of potential [duplication and configuration
+  issues](https://github.com/mrdoob/three.js/pull/29404).
+</Tip>
+## WebGPU
+
+To use Three.js's WebGPU renderer, import it and then initialize it within your
+`<Canvas>`'s `createRenderer` prop.
+
+```svelte title="App.svelte" {4}+ {8-14}+
+<script>
+  import Scene from './Scene.svelte'
+  import { Canvas } from '@threlte/core'
+  import * as THREE from 'three/webgpu'
```

Comment:
Would recommend against wildcard imports in the docs to encourage best practices for tree shaking:
```suggestion
  import { WebGPURenderer } from 'three/webgpu'
```